src/models/curriculum/unit.ts

```diff
@@ -13,6 +15,7 @@ import { NavTabsConfigModel } from "../stores/nav-tabs";
 import { SettingsMstType } from "../stores/settings";
 import { IBaseStores } from "../stores/stores";
 import { UnitConfiguration } from "../stores/unit-configuration";
+import { ProblemModelType } from "./problem";
```

Comment:
I would move this ⬆️ to be with the `./investigation` and `./section` imports above